Docs: additional caching documentation (#31925)
grafana/grafana: docs/sources/enterprise/query-caching.md

## Enable query caching

To enable and configure query caching, please refer the the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#query-caching" >}}).

## Skip caching per request

> Requests that contain the header `X-Cache-Skip` with any value will be skipped. This can be particularly useful when debugging datasource queries using cURL.

Reviewer: ```suggestion
Requests that contain the header `X-Cache-Skip` with any value are skipped. This can be particularly useful when debugging data source queries using cURL.
```

Reviewer: By "skipped" do you mean they are not cached? If so, then we should say that. Also, if you can state this in an active way, then the sentence would be clearer.

Author: Correct.

:+1: I'll update that
